inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/DataStreamServiceImpl.java

```diff
@@ -575,6 +576,16 @@ public int selectCountByGroupId(String groupId) {
         return count;
     }
 
+    @Override
+    public List<DataStreamTopicVO> getTopicList(String groupId) {
+        LOGGER.debug("begin bo get topic list by group id={}", groupId);
+        Preconditions.checkNotNull(groupId, BizConstant.GROUP_ID_IS_EMPTY);
+
+        List<DataStreamTopicVO> topicList = streamMapper.selectTopicList(groupId);
+        LOGGER.debug("success to get topic list");
```

Comment:
add more request info in debug is better。eg. groupid

Reply:
added more info.